src/components/AddBareMetalHosts/AddBareMetalHosts.tsx

```diff
@@ -1,14 +1,12 @@
-import { ButtonVariant, PageSectionVariants } from '@patternfly/react-core';
+import { ButtonVariant, Card, CardBody, CardTitle, Title, Toolbar } from '@patternfly/react-core';
 import React from 'react';
 import { useDispatch } from 'react-redux';
 import { Cluster, getErrorMessage, handleApiError, installHosts } from '../../api';
 import { addAlert } from '../../features/alerts/alertsSlice';
 import { updateCluster } from '../../features/clusters/currentClusterSlice';
-import ClusterToolbar from '../clusters/ClusterToolbar';
-import ClusterBreadcrumbs from '../clusters/ClusterBreadcrumbs';
 import { isKnownHost } from '../hosts/utils';
```

Comment:
renaming this utility function to `hasKnownHosts` would greatly help code readability. I had to look at the function to know what exactly it does.

Reply:
done